src/lumen_anndata/views.py

```diff
@@ -1,12 +1,50 @@
 import holoviews as hv
+import param
 
 from hv_anndata import ManifoldMap
+from lumen.ai.memory import _Memory, memory
 from lumen.views import View
 
 
 class ManifoldMapPanel(View):
+
+    memory = param.ClassSelector(class_=_Memory, default=None, doc="""
```

Comment:
`View` classes should not have access to `memory` imo, we should try to maintain a separation of concerns.